server/planning/content_api/events/event.py

```diff
@@ -63,10 +56,5 @@ async def publish_async(self, item, subscribers=None) -> None:
             MongoIndexOptions(name="template", keys=[("template", 1)], unique=False),
         ],
     ),
-    elastic=ElasticResourceConfig(prefix=ELASTIC_PREFIX),
-    rest_endpoints=RestEndpointConfig(
-        resource_methods=["GET"],
-        item_methods=["GET"],
-        enable_cors=True,
-    ),
+    elastic=ElasticResourceConfig(),
```

Comment:
`ELASTIC_PREFIX` is required, so that the data is stored separately to the rest of Superdesk